src/constants/ow2_constants.opy

```diff
@@ -79,4 +79,8 @@
 #!define OW2_WINSTON_ULT_COST 1850
 
 # Wrecking ball
-#!define OW2_WRECKING_BALL_PILEDRIVER_CD 8
\ No newline at end of file
+#!define OW2_WRECKING_BALL_PILEDRIVER_CD 8
+
+# Cassidy
+#!define OW2_CASSIDY_ROLL_DAMAGE_REDUCTION 50
+#!define OW2_CASSIDY_DEADEYE_DAMAGE_REDUCTION 40
```

Comment:
Rename Cassidy to Mccree becasue OverPy uses Mccree to refer to Cassidy.

Also move this section of code to line ~66 (after Bastion constants) to keep the hero constants alphabetical.